src/Shared/ContainerImageTags.cs

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Components.Common;
+
+internal static class ContainerImageTags
+{
+    public static (string image, string tag) Redis = ("redis", "7.2.4");
```

Comment:
```suggestion
    public static (string Image, string Tag) Redis = ("redis", "7.2.4");
```

The naming conventions are to use PascalCase because these are just like properties.